src/OrleansRuntime/GrainDirectory/GrainDirectoryPartition.cs

```diff
@@ -364,21 +376,58 @@ internal void RemoveGrain(GrainId grain)
         internal AddressesAndTag LookUpActivations(GrainId grain)
```

Comment:
Since we are staring at this code right now anyway: as I remarked in #1981: the tag (some kind of version number of the directory entry for a particular grain) is never actually really needed for anything (I suspect it had a purpose in the past at some point). We can save space and time by removing it everywhere in the code.